doozer/doozerlib/runtime.py

```diff
@@ -686,36 +696,8 @@ def initialize_logging(self):
         else:
             log_level = logging.INFO
 
-        default_log_formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
-
-        root_logger = logging.getLogger()
-        root_logger.setLevel(logging.WARN)
-        root_stream_handler = logging.StreamHandler()
-        root_stream_handler.setFormatter(default_log_formatter)
-        root_logger.addHandler(root_stream_handler)
-
-        # If in debug mode, let all modules log
-        if not self.debug:
-            # Otherwise, only allow children of ocp to log
-            root_logger.addFilter(logging.Filter("ocp"))
-
-        # Get a reference to the logger for doozer
-        self.logger = logutil.get_logger()
-        self.logger.propagate = False
-
-        # levels will be set at the handler level. Make sure master level is low.
-        self.logger.setLevel(logging.DEBUG)
-
-        main_stream_handler = logging.StreamHandler()
-        main_stream_handler.setFormatter(default_log_formatter)
-        main_stream_handler.setLevel(log_level)
-        self.logger.addHandler(main_stream_handler)
-
-        debug_log_handler = logging.FileHandler(self.debug_log_path)
-        # Add thread information for debug log
-        debug_log_handler.setFormatter(logging.Formatter('%(asctime)s %(levelname)s (%(thread)d) %(message)s'))
-        debug_log_handler.setLevel(logging.DEBUG)
-        self.logger.addHandler(debug_log_handler)
+        logutil.init_logging(log_level, self.debug_log_path)
```

Comment:
I don't see `init_logging` defined anywhere, you meant `setup_logging`?

Reply:
Yes, I renamed this function before committing but forgot to update the callers.
Now updated.